sympy/physics/mechanics/tests/test_functions.py

```diff
@@ -198,3 +198,26 @@ def test_gravity():
     for i in range(len(l)):
         for j in range(len(l[i])):
             assert forceList[i][j] == l[i][j]
+
+
+def test_center_of_mass():
```

Comment:
Can you add a comment explaining what exactly this is testing and reference an issue where the bug was found?

Reply:
Is this an example that comes from one of the autolev texts?

Reply:
This is the test for the center of mass functions I had previously added but did not include a test for. There is no issue associated with it.

Reply:
The example isn't from an Autolev text.

Reply:
Ok, please add a comment to the code explaining what it is testing and reference where you found the bug.